dts/bindings/gpio/gaisler,grgpio.yaml

```diff
@@ -0,0 +1,19 @@
+description: GRLIB GRGPIO
+
```

Comment:
Can we add a reference to the doc link you provided here.

Reply:
It is added now. Thanks for the suggestion.